src/components/widgets/Button.js

```diff
@@ -18,7 +26,13 @@ class Button extends Component {
       classes += ` button--default`;
     }
 
-    classes += ` ${className}`;
+    if (group) {
+      classes += ' button--menu';
```

Comment:
In the name of keeping our widgets dumb perhaps instead of adding logic that knows about groups in a the button widget maybe we should just pass in a className. So th parent would supply `button--menu` and the child would simply render whatever class it is passed.